tests/deployment_utils.py

```diff
@@ -2,10 +2,11 @@
 
 import getpass
 import os
+import shutil
+import pathlib
```

Comment:
We have several places in repo where `from pathlib import Path` is used. Maybe we want to standardize Path usage in this very commit?